step-templates/swaggerhub-post-api.json

```diff
@@ -0,0 +1,117 @@
+        "Label": "SwaggerHub API Key",
+        "HelpText": "Your SwaggerHub API key",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Sensitive"
+        }
+      },
+      {
+        "Id": "7047922F-8AA3-476F-B2E6-89E9C6F0DB3F",
+        "Name": "shpOwner",
+        "Label": "Owner",
+        "HelpText": "API owner (user or organization, case-sensitive)",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "BBE8AB7A-2BFE-4413-ACFF-384F6B6F953C",
+        "Name": "shpApi",
+        "Label": "Api Name",
+        "HelpText": "API name (case-sensitive)",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "409EAF93-FF98-4430-89CD-19C30E1CC792",
+        "Name": "shpIsPrivate",
+        "Label": "Is Private",
+        "HelpText": "Defines whether the API has to be private",
+        "DefaultValue": "False",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Checkbox"
+        }
+      },
+      {
+        "Id": "3643C071-B51F-4611-900C-FAB22B4C5698",
+        "Name": "shpVersion",
+        "Label": "Version",
+        "HelpText": "API version",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "25550589-4208-476C-B581-20EF0A4F1453",
+        "Name": "shpOas",
+        "Label": "OAS Version",
+        "HelpText": "The OpenApi Specification (OAS) version.",
+        "DefaultValue": "2.0",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Select",
+          "Octopus.SelectOptions": "2.0|2.0\n3.0.0|3.0.0\n3.0.1|3.0.1"
+        }
+      },
+      {
+        "Id": "AF1EF8F1-A829-4D40-86B1-9D18D909037E",
+        "Name": "shpDefinition",
+        "Label": "Definition",
+        "HelpText": "The Swagger definition, or the path to the file that contains the definition, of this API",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "MultiLineText"
+        }
+      },
+      {
+        "Id": "F99E5B74-55BF-4C91-83E6-E6AC17284F4C",
+        "Name": "shpContentType",
+        "Label": "Definition Content Type",
+        "HelpText": "Definition Content Type",
+        "DefaultValue": "application/json",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Select",
+          "Octopus.SelectOptions": "application/json|application/json\napplication/yaml|application/yaml"
+        }
+      },
+      {
+        "Id": "3790F1C7-3FA8-4908-B4C3-F96DCD02B96D",
+        "Name": "shpForce",
+        "Label": "Force",
+        "HelpText": "Force update",
+        "DefaultValue": "False",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Checkbox"
+        }
+      }
+    ],
+    "SpaceId": "Spaces-1",
```

Comment:
Sorry for the delay.  Could you remove the SpaceId parameter?

Reply:
No worries!  Thanks for the feedback!

SpaceId removed.